test/noticed_test.rb

```diff
@@ -167,4 +171,10 @@ class Noticed::Test < ActiveSupport::TestCase
       With5MinutesDelay.new.deliver(user)
     end
   end
+
+  test 'asserts delivery is queued with different queue' do
+    assert_enqueued_with(queue: 'custom') do
+      WithCustomQueue.new.deliver(user)
```

Comment:
I think you need this to be `deliver_later` to fix your test. `deliver` won't enqueue anything cuz it runs immediately.

Reply:
Oh, derp, good catch. Was copying off `delay`, but delay will always be done async.

Reply:
Thanks!